packages/core/dist/GondelComponent.d.ts

```diff
@@ -10,6 +17,10 @@ export interface GondelComponent {
     sync?(): void;
 }
 export declare class GondelBaseComponent implements GondelComponent {
+    /**
```

Comment:
With this change Gondel will only be usable together with a bundler as it enforces to extend from BaseComponent.

I am not 100% sure if this is an issue or not - what do you think?

Also I am not sure if a static value could cause problems with our hot module replacement.  
Do you see an issues there?